pkg/build/path_rewriter.go

```diff
@@ -22,23 +22,20 @@ import (
 // RelativePathRewriter rewrites paths when the paths are relative paths.
 type RelativePathRewriter struct{}
 
-// MakeAbs rewrites file paths if the path is relative, from the
+// AbsWithParent rewrites file paths if the path is relative, from the
```

Comment:
This whole file should just be plain vanilla functions.  You then have to instance this and struct shove.  That can be avoided.

Reply:
Done